doc/source/snippets/codespell-ignore.rst

```diff
@@ -1,4 +1,4 @@
-To ignore a word, add it to ``.codespell/ignore_words.txt``. See an example `here <https://github.com/diffpy/diffpy.fourigui/blob/main/.codespell/ignore_words.txt>`_.
+To ignore a word, add it to ``.codespell/ignore_words.txt``. You can find an example file here: https://github.com/diffpy/diffpy.fourigui/blob/main/.codespell/ignore_words.txt.
```

Comment:
`sphinx` doesn't like using "here" and also I realized that it's much easier to maintain when we simply present the URL.